packages/rn-tester/android/app/build.gradle

```diff
@@ -254,7 +264,6 @@ if (enableCodegen) {
         defaultConfig {
             externalNativeBuild {
                 ndkBuild {
-                    abiFilters "armeabi-v7a", "x86", "x86_64", "arm64-v8a"
```

Comment:
This is the default already according to https://developer.android.com/ndk/guides/abis. This will use the project wide config added a few lines up.